### PR TITLE
Remove build warnings from console-ui

### PR DESCRIPTION
**Pull request by anukritijha**
### Type of change

- Refactoring

### Description

Remove build warnings from console - UI.

### CODE REVIEW

File: console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
           value={TypeAheadMessage.MORE_CHAR_REQUIRED}
           isDisabled={true}
           key="2"
-        />
+        />,
```
Comment (ajaypratap003) [inline console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx:58]: Is there any specific reason for adding comma?

File: console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx [hunk 2]
```diff
@@ -77,7 +77,7 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
               <SelectOption
                 value={TypeAheadMessage.NO_RESULT_FOUND}
                 isDisabled={true}
-              />
+              />,
```
Comment (ajaypratap003) [inline console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx:80]: Is there any specific reason for adding comma?

File: console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx [hunk 3]
```diff
@@ -88,7 +88,7 @@ const TypeAheadSelect: React.FunctionComponent<ITypeAheadSelectProps> = ({
         value={TypeAheadMessage.MORE_CHAR_REQUIRED}
         key="1"
         isDisabled={true}
-      />
+      />,
```
Comment (ajaypratap003) [inline console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx:91]: Is there any specific reason for adding comma?

File: console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx (hunk 1 above)
Comment (anukritijha) [reply console/console-init/ui/src/components/TypeAheadSelect/TypeAheadSelect.tsx:58]: This has automatically added because of prettier.

**Pull request closed without merging**